app/views/hackdays/_projects.html.erb

```diff
@@ -0,0 +1,19 @@
+	<div class = "row">
+		<!-- row for past hackday -->
+		<div class = "col-md-6 col-md-offset-3">
+			<h1> Projects </h1>
+			<h4> votes left: <%= (3-cookies[:"current_votes_#{@hackday.id}"].to_i)  %> </h4>
```

Comment:
might be nicer to define this as an @ variable in the controller rather than do this logic in the view.